python/lsst/ip/isr/overscan.py

```diff
@@ -268,7 +486,7 @@ def getImageArray(self, image):
         return calcImage
 
     @staticmethod
-    def transpose(imageArray):
+    def transpose(imageArray, isTransposed=False):
```

Comment:
This seems like a meaningless addition to me: telling the function whether you want it to do something. If you don't want it to do anything, don't call it in the first place.

Reply:
I've dropped the method, as it's trivial without that logic.